development/nautobot_config.py

```diff
@@ -2,11 +2,20 @@
 # pylint: disable=invalid-envvar-default
 import os
 import sys
+import json
 
 from nautobot.core.settings import *  # noqa: F403
 from nautobot.core.settings_funcs import parse_redis_connection
 
 
+def _get_env(name: str, default="") -> str:
+    return os.getenv(f"NAUTOBOT_SSOT_{name}", default)
+
+
+def _get_bool_env(name: str, default=False) -> bool:
```

Comment:
Several reasons :-)

- `is_truthy` is using `strtobool`, that's deprecated in Python 3.10, will be removed.
- `get_bool_env` does more that `is_truthy` (reads from env, prefixing `NAUTOBOT_SSOT` here), IMHO easier to implement locally using standard libs over custom implementation.

Reply:
I can rename these functions to `_get_ssot_env` or `_get_ssot_bool_env`, would that be more clear?

Reply:
I wasn't aware of the deprecation. If that's the case, it might make more sense to update that function in the core app to your method as I guarantee there are a lot of configurations in use out there using `is_truthy` that's going to break in the future. Better to fix that and continue using it instead of creating some replacement.

Reply:
Fixed as proposed.